template/cordova/lib/target-list.js

```diff
@@ -17,20 +17,21 @@
        under the License.
 */
 
-var devices = require('./package'),
+var deploy  = require('./deployment'),
     args = process.argv.slice(2);
 
 // help/usage function
 function help() {
     console.log('');
-    console.log('Usage: node target-list.js  [ --emulators | --devices | --started_emulators | --all ]');
+    console.log('Usage: node target-list.js [--win10] [ --emulators | --devices | --started_emulators | --all ]');
+    console.log('    --win10             : Chooses to list Windows 10 devices (Windows 8.1 is default).');
     console.log('    --emulators         : List the possible target emulators availible.');
-    console.log('    --devices           : List the possible target devices availible. *NOT IMPLEMENTED YET*');
+    console.log('    --devices           : List the possible target devices availible.');
     console.log('    --started_emulators : List any started emulators availible. *NOT IMPLEMENTED YET*');
     console.log('    --all               : List all available devices');
     console.log('examples:');
     console.log('    node target-list.js --emulators');
-    console.log('    node target-list.js --devices');
+    console.log('    node target-list.js --win10 --devices');
```

Comment:
Yeah, this is a complicated question, and I'm not sure what the right answer is right now.

The Windows 8.1 tool actually enumerates Windows 10 emulators, but it can't deploy to them.  Conversely, the Windows 10 tool does not presently enumerate Windows 10 emulators (because it also can't deploy to them).  The Windows 8.1 tool is fast, the Windows 10 tool is slow (it does network probing).  We could possibly display both, but most likely we would end up simply having the following list, for example:

0. Device (device) [Windows 8.1 tools]
1. Mobile 4 inch 512MB 10.0.10176.0 (emulator) [Windows 8.1 tools]
2. Mobile 4 inch 1GB 10.0.10176.0 (emulator) [Windows 8.1 tools]
3. Mobile 5 inch 1GB 10.0.10176.0 (emulator) [Windows 8.1 tools]
4. Mobile 6 inch 2GB 10.0.10176.0 (emulator) [Windows 8.1 tools]
5. Windows Phone 8.1 4 inch 512MB (emulator) [Windows 8.1 tools]
6. Windows Phone 8.1 4 inch 1GB (emulator) [Windows 8.1 tools]
7. Windows Phone 8.1 5 inch 1GB (emulator) [Windows 8.1 tools]
8. Windows Phone 8.1 6 inch 2GB (emulator) [Windows 8.1 tools]
9. Lumia 1520 (RM-940) (device) [Windows 10 tools]

0 and 9 end up being the same thing, which is problematic.  I just don't know which is better: to repeat devices, or to have an option that picks "is my target device Windows 8.1 or Windows 10?"  I landed on the option choice, for compatibility reasons (opt into the new behavior).